tests/test_cachito_config.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from unittest.mock import patch
+
+import pytest
+
+from cachito.web.config import validate_cachito_config
+from cachito.errors import ConfigError
+
+
+@patch('os.path.isdir', return_value=True)
+def test_validate_cachito_config_success(mock_isdir, app):
+    validate_cachito_config(app.config)
+    mock_isdir.assert_any_call('/tmp/cachito-archives/bundles')
+
+
+@patch('os.path.isdir', return_value=False)
+@pytest.mark.parametrize('variable_name', (
+    'CACHITO_BUNDLES_DIR', 'CACHITO_LOG_LEVEL', 'CACHITO_MAX_PER_PAGE', 'CACHITO_LOG_FORMAT'))
+def test_validate_cachito_config_failure(mock_isdir, app, variable_name):
+    expected = f'The configuration "{variable_name}" must be set'
+    temp = app.config[variable_name]
+    app.config[variable_name] = None
```

Comment:
Could you use https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict? This is a safer way of doing this.

Reply:
really cool!